packages/ui-library/src/components/forms/textarea/index.ts

```diff
@@ -186,29 +189,24 @@ export class BlrTextarea extends LitElement {
             })
           : nothing}
       `;
-
+      //prettier-ignore
```

Comment:
I wanted to suggest adding an additional comment here to explain the issue, but I think I found a better solution, with which you can remove this again, see the next comment.

```suggestion

```